index.Rmd

```diff
@@ -53,8 +53,15 @@ The course will cover:
 - Guidance on how to update a DMS plan after submission or funding
 - General tips and frequently asked questions
 
-```{r, echo=FALSE, fig.alt='Pathway of chapters covered in this course.', out.width = '100%'}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1luFoDzF6aDJEebbL6iWoJ_s8s9nQnaWLL5jghbmWdak/edit#slide=id.g16b04bd80e5_1_123")
+
+```{r, echo=FALSE, fig.alt='Main topics covered in this course: Introducing the DMS policy, Creating a DMS plan, DMS submission process.', out.width = '100%'}
```

Comment:
I feel like this image belongs right before "Motivation" H2. Thoughts?

Reply:
I think @avahoffman is right.

Reply:
so you mean as the first thing for the next section?